config/default.go

```diff
@@ -145,6 +149,7 @@ MaxRequestsPerIPAndSecond = 10
 
 [BridgeL1Sync]
 DBPath = "{{PathRWData}}/bridgel1sync.sqlite"
+BlockFinality = "FinalizedBlock"
```

Comment:
Didn't we want to have the `LatestBlock` as finality, and report them as well (and that's why we are reverting the reorg detector) WDYT?

```suggestion
BlockFinality = "LatestBlock"
```

Reply:
by default we want this to be `FinalizedBlock` and only apps team will edit this on their aggkit instance to `LatestBlock`